build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -93,30 +100,36 @@ export function fromInternalScheme(
 
  * @param publicVersion - The public version.
  * @param version - The internal version.
+ * @param allowPrereleases - If true, allow prerelease Fluid internal versions.
```

Comment:
Do we have somewhere full matrix of valid versions (covering prereleases)? I don't see them in the tests below. or I'm not reading them correctly :)

Reply:
As far as I can tell the test cases in internalVersionScheme.test.ts cover this method when `allowPrereleases` is both true and false.